server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const path = require('path')
+
+const app = express()
+
+const PORT = 9876
+
+app.listen(PORT, function() {
+  console.log('server is running and listening on port', PORT)
+})
```

Comment:
lets put the listen at the bottom of this file - to make sure the config (app.use) happens first.

Note also, that we will need an index.js which we cant export without starting a server listening (for testing an API we're going to build)